### DISCUSSION TITLE
Does Table draggable drag-n-drop work on touch screens?

### DISCUSSION THREAD

**Discussion by lownlazy** [buefy/buefy#3623] [Q&A] [answered]
From what I can see it doesn't but I hope I am missing something. I am using Chrome on Android and using (for example) this site: https://codepen.io/yxngl/pen/oRzdQe  
With a desktop and mouse the rows turn purple when clicked and drag-n-drop as expected. 
In touch screens there seems to be no drag events fired at all, the table behaves as if 'draggable' is set to false. 

Is there some way to enable draggable on a touch screen?

Many thanks for your help with this and the excellent Buefy software.

**Comment by gruelt** [comment]
Hi , you're not alone , I can't use drag & drop on touch screens .
Nobidy since 2021 has found a way to make this work ?

Thanks

**Reply by wesdevpro** [reply]
This is interesting.

**Reply by kikuomax** [reply, marked as answer]
It is now supported since v0.9.27.
